pandas/tests/test_categorical.py

```diff
@@ -2934,26 +2933,28 @@ def test_value_counts_with_nan(self):
             s = pd.Series(pd.Categorical(["a", "b", "a"],
```

Comment:
this is an incorrect construction, categories should not be ``nan`` by definition